app/packages/rust_sdk/lib/acter_flutter_sdk.dart

```diff
@@ -317,10 +318,11 @@ class ActerSdk {
     return client;
   }
 
-  Future<void> logout() async {
+  Future<bool> logout() async {
     // remove current client from list
-    var client = _clients.removeAt(_index);
+    final client = _clients.removeAt(_index);
     _index = _index > 0 ? _index - 1 : 0;
+    print("Remainig clients $_clients");
     await _persistSessions();
```

Comment:
oddly this seems to write the wrong sessions... despite us having removed the item :( ...

Reply:
appears to have been a problem of having more than one client for some reason.